packages/eslint-config-ciscospark/react.js

```diff
@@ -0,0 +1,273 @@
+      0
+    ],
+    'react/display-name': [
+      2
+    ],
+    'react/forbid-component-props': [
+      2
+    ],
+    'react/forbid-prop-types': [
+      2,
+      {
+        forbid: [
+          'style'
+        ]
+      }
+    ],
+    'react/no-children-prop': [
+      2
+    ],
+    'react/no-danger': [
+      2
+    ],
+    'react/no-danger-with-children': [
+      2
+    ],
+    'react/no-deprecated': [
+      2
+    ],
+    'react/no-did-mount-set-state': [
+      2
+    ],
+    'react/no-did-update-set-state': [
+      2
+    ],
+    'react/no-direct-mutation-state': [
+      2
+    ],
+    'react/no-find-dom-node': [
+      2
+    ],
+    'react/no-is-mounted': [
+      2
+    ],
+    'react/no-multi-comp': [
+      2
+    ],
+    'react/no-render-return-value': [
+      2
+    ],
+    'react/no-set-state': [
+      2
+    ],
+    'react/no-string-refs': [
+      2
+    ],
+    'react/no-unescaped-entities': [
+      2
+    ],
+    'react/no-unknown-property': [
+      2
+    ],
+    'react/no-unused-prop-types': [
+      2
+    ],
+    'react/prefer-es6-class': [
+      2
+    ],
+    'react/prefer-stateless-function': [
+      2
+    ],
+    'react/prop-types': [
+      2
+    ],
+    'react/react-in-jsx-scope': [
+      2
+    ],
+    'react/require-optimization': [
+      2
+    ],
+    'react/require-render-return': [
+      2
+    ],
+    'react/self-closing-comp': [
+      2
+    ],
+    'react/sort-comp': [
+      2
+    ],
+    'react/sort-prop-types': [
+      2
+    ],
+    'react/style-prop-object': [
+      2
+    ],
+    //
+    // JSX-specific rules
+    //
+    'react/jsx-boolean-value': [
+      2
+    ],
+    'react/jsx-closing-bracket-location': [
+      2
+    ],
+    'react/jsx-curly-spacing': [
+      2,
+      'never'
+    ],
+    'react/jsx-equals-spacing': [
+      2,
+      'never'
+    ],
+    'react/jsx-filename-extension': [
+      2,
+      {
+        extensions: [
+          '.js'
+        ]
+      }
+    ],
+    'react/jsx-first-prop-new-line': [
+      2,
+      'multiline'
+    ],
+    'react/jsx-handler-names': [
+      2
+    ],
+    'react/jsx-indent': [
+      2,
+      2
+    ],
+    'react/jsx-indent-props': [
+      2,
+      2
+    ],
+    'react/jsx-key': [
+      2
+    ],
+    'react/jsx-max-props-per-line': [
+      2,
+      {
+        maximum: 3
+      }
+    ],
+    'react/jsx-no-bind': [
+      2
+    ],
+    'react/jsx-no-comment-textnodes': [
+      2
+    ],
+    'react/jsx-no-duplicate-props': [
+      2
+    ],
+    'react/jsx-no-literals': [
+      2
+    ],
+    'react/jsx-no-target-blank': [
+      2
+    ],
+    'react/jsx-no-undef': [
+      2
+    ],
+    'react/jsx-pascal-case': [
+      2
+    ],
+    'react/jsx-sort-props': [
+      2
+    ],
+    'react/jsx-space-before-closing': [
+      2
+    ],
+    'react/jsx-uses-react': [
+      2
+    ],
+    'react/jsx-uses-vars': [
+      2
+    ],
+    'react/jsx-wrap-multilines': [
+      2
+    ],
+    //
+    // jsx-a11y
```

Comment:
Should we break out the `jsx-a11y` rules in case we want to do some sort of server side, non-a11y, react projects? Now that I type that out, doesn't seem to be worthwhile, but feel free to respond anyway 😄